vignettes/template-doc.Rmd

```diff
@@ -19,15 +19,15 @@ will find the following assets for recording your projects metadata:
 
 * methods.docx
 * abstract.docx
-* metadata.xlsx
+* project-metadata.xlsx
```

Comment:
these changes look good! Please rebuild pkgdown site so that the article is updated on the site.